src/iris/api.py

```diff
@@ -18,7 +18,9 @@
 from jinja2.sandbox import SandboxedEnvironment
 from urlparse import parse_qs
 import ujson
-from falcon import HTTP_200, HTTP_201, HTTP_204, HTTPBadRequest, HTTPNotFound, HTTPUnauthorized, HTTPForbidden, HTTPFound, HTTPInternalServerError, API
+from falcon import (before, HTTP_200, HTTP_201, HTTP_204, HTTPBadRequest,
```

Comment:
I think it'd be cleaner to use `falcon.before` instead of using a global ambiguous `before`